lib/pkgr/builder.rb

```diff
@@ -284,17 +287,23 @@ def distribution
       @distribution ||= Distributions.current(config)
     end
 
-    # List of available buildpacks for the current distribution.
-    def buildpacks
-      distribution.buildpacks
-    end
-
-    # Buildpack detected for the app, if any.
-    def buildpack_for_app
+    # Buildpacks detected for the app, if any. If multiple buildpacks are explicitly specified, all are used
+    def buildpacks_for_app
       raise "#{source_dir} does not exist" unless File.directory?(source_dir)
-      @buildpack_for_app ||= buildpacks.find do |buildpack|
-        buildpack.setup(config.edge, config.home)
-        buildpack.detect(source_dir)
+      @buildpacks_for_app ||= begin
+        mode, buildpacks = distribution.buildpacks
+        case mode
+        when :custom
+          buildpacks.find_all do |buildpack|
+            buildpack.setup(config.edge, config.home)
+            buildpack.detect(source_dir)
+          end
+        else
+          [buildpacks.find do |buildpack|
+            buildpack.setup(config.edge, config.home)
+            buildpack.detect(source_dir)
+          end].compact
```

Comment:
What is the purpose of this distinction here? `buildpacks` are always a list, can be also read from https://github.com/crohr/pkgr/blob/master/data/buildpacks/* according to the system version you are building, but leaving this code you have preserved to run only the first buildpack from the list.
After the introduction of https://github.com/crohr/pkgr/commit/8ec93e9789bfc01e75cb437a983060d8a153baae I'm using my own set of configuration. I rather want to modify the bunch of settings in data-dir rather than pass another argument to the pkgr binary.